region/lowernorfair/east.json

```diff
@@ -1407,38 +1430,52 @@
               "id": 4,
               "requires": [
                 {"or": [
-                  "heatProof",
-                  "canHeatRun"
+                   "heatProof",
+                   "canHeatRun"
                 ]},
-                "canUsePowerBombs",
-                {"or": [
-                  "canWalljump",
-                  {"cost": [
-                    {"heatFrames": 200}
-                  ]}
-                ]},
-                {"and" :[
-                  {"cost": [
-                    {"heatFrames": 800}
-                  ]},
-                  {"or": [
-                    {"cost": [
-                      {"enemyDamage": [{
-                        "enemy": 79,
-                        "type": "contact",
-                        "hits": 1
-                      }]}
+                {"and": [
```

Comment:
This `and` is redundant because elements at the first level of a `requires` array are implicitly `and`ed.

Looks perfect beyond that, though. It looks like the XRay climb here is being added as an alternative to jumping up and using the PB, but not to dealing with the Dessgeega at the top.

Reply:
To fix the camera, you have to jump a bunch where the one bomb block is so you still need to deal with the dessgeegas.  Actually, with morph bombs you wouldn't have to fix the camera.  Then the enemies wouldn't be interactable.

Reply:
I think leaving it in is fine though. That way the logic can't expect you to x-ray climb then go right without dealing with it.